mock/cpp_mock_scenarios/src/cpp_scenario_node.cpp

```diff
@@ -18,11 +18,9 @@
 namespace cpp_mock_scenarios
 {
 CppScenarioNode::CppScenarioNode(
-  const std::string & node_name, const std::string & map_path,
-  const std::string & lanelet2_map_file, const std::string & scenario_filename, const bool verbose,
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.